bazel/tools/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
```

Comment:
Should these go under bazel/rules instead? Or even the other way around (as what seem to have over there currently seems to be a macro and a target for a tool)? Let's just agree on some convention and stick to it.

Reply:
golden_file_test definitly in rules.
I prefer tools for the things I don't think normal people should use.
I'll move g_f_t and lock down the other.